playground/src/hoc/with-error-boundary.usage.tsx

```diff
@@ -1,16 +1,33 @@
 import * as React from 'react';
+import {useState} from 'react';
 
 import { withErrorBoundary } from '../hoc';
 import { ErrorMessage } from '../components';
 
 const ErrorMessageWithErrorBoundary =
   withErrorBoundary(ErrorMessage);
 
-const BrokenButton = () => (
-  <button type="button" onClick={() => { throw new Error(`Catch me!`); }}>
-    {`Throw nasty error`}
-  </button >
-);
+const BrokenButton = () => {
+  const [throwError, setThrowError] = useState(false);
```

Comment:
Please rename `throwError` to `shouldRenderBrokenComponent`

Reply:
The line became quite long because of `[shouldRenderBrokenComponent, setShouldRenderBrokenComponent]`
so I think `setShouldBreak `would be a better name.